q2_alignment/_filter.py

```diff
@@ -69,6 +69,10 @@ def mask(alignment: skbio.TabularMSA, max_gap_frequency: float=0.05,
     if min_conservation < 0.0 or min_conservation > 1.0:
         raise ValueError('min_conservation out of range [0.0, 1.0]: %f' %
                          min_conservation)
+    # check that input alignment is not empty
+    if alignment.shape.position == 0:
+        raise ValueError('Input alignment is empty (i.e., there are zero '
+                         'sequences or positions in the input alignment).')
```

Comment:
Technically there can be sequences in the TabularMSA with no positions. Not worth fixing the error message though